### DISCUSSION TITLE
Any Solution?

### DISCUSSION THREAD

**Discussion by sayeedajmal** [AkashSingh3031/The-Complete-FAANG-Preparation#123] [General]
What Can  I Do? 
![Screenshot_2021-07-02_01_11_56](https://user-images.githubusercontent.com/76242275/124138912-bc164580-daa4-11eb-9f1c-535bf62c48b2.png)

**Comment by AkashSingh3031** [comment]
If you add explanation, then send pull request

**Reply by sayeedajmal** [reply]
okey!
